dev/node_app/package.json

```diff
@@ -11,7 +11,7 @@
   "license": "ISC",
   "dependencies": {
     "express": "4.17.1",
-    "mattermost-redux": "github:mattermost/mattermost-redux#5fa256e1f5ae6ba237a276e35ec23a4c1f73af7b",
+    "mattermost-redux": "5.33.1",
```

Comment:
FYI we're moving away from `mattermost-redux` for node apps, as all wanted things should be available in new packages `@mattermost/types` and `@mattermost/client` now. It takes way less time to download!!

Reply:
Thanks @mickmister! I tried updating my dependencies like so:

```
"dependencies": {
    "express": "4.17.1",
    "@mattermost/types": "6.7.0-0",
    "@mattermost/client": "6.7.0-0",
    "node-fetch": "2.6.1"
  },
```

Then in `app.ts` I changed the imports to the following:

```
import {AppBinding, AppCallRequest, AppCallResponse, AppForm, AppManifest} from '@mattermost/types/apps';
import {Post} from '@mattermost/types/posts';
import {Channel} from '@mattermost/types/channels';

import {Client4} from '@mattermost/client';
```

However, when trying to run the app, I receive this error:

```
/Users/andrew/techne/__REPOS/mattermost-plugin-apps/dev/node_app/node_modules/@mattermost/client/lib/index.js:3
export { default as Client4 } from './client4';
^^^^^^

SyntaxError: Unexpected token 'export'
```

I believe, based on [this part of the README](https://github.com/mattermost/mattermost-webapp/tree/master/packages#adding-a-new-subpackage) that I need to modify `tsconfig.json` as well, but I'm not sure how to do so for this purpose. Can you help me resolve this?

Reply:
What if you try

```ts
import Client4 from '@mattermost/client/client4';
```

Reply:
Just tried that, my error is:

```
Error: Cannot find module '@mattermost/client/client4'
```

Perhaps that is an alias that needs to be set in `tsconfig.json`?

Reply:
I think the full path would actually be `@mattermost/client/lib/client4`, but I'm not sure if that will work either.

Reply:
@mickmister If I try `@mattermost/client/lib/client4` I get `Cannot use import statement outside a module`.